ethcore/src/migrations/v11.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+
+//! Adds a seventh column for node information.
+
+use util::kvdb::Database;
+use util::migration::{Batch, Config, Error, Migration, Progress};
+use std::sync::Arc;
+
+/// Copies over data for all existing columns.
+#[derive(Default)]
+pub struct ToV11(Progress);
+
+
+impl Migration for ToV11 {
```

Comment:
But you can just skip the migration at all right? (just alter `v10` to return more columns and that's it) Since if I understand correctly opening a database specifying more columns should work fine.